lib/matplotlib/pyplot.py

```diff
@@ -1311,19 +1311,19 @@ def subplot_tool(targetfig=None):
     """
     Launch a subplot tool window for a figure.
 
-    A :class:`matplotlib.widgets.SubplotTool` instance is returned.
+    A `matplotlib.widgets.SubplotTool` instance is returned.
     """
     if targetfig is None:
         targetfig = gcf()
-
-    with rc_context({'toolbar': 'None'}):  # No nav toolbar for the toolfig.
-        toolfig = figure(figsize=(6, 3))
-    toolfig.subplots_adjust(top=0.9)
-
-    if hasattr(targetfig.canvas, "manager"):  # Restore the current figure.
-        _pylab_helpers.Gcf.set_active(targetfig.canvas.manager)
-
-    return SubplotTool(targetfig, toolfig)
+    with rc_context({"toolbar": "none"}):  # No navbar for the toolfig.
```

Comment:
would it make more sense for this to be in backend bases and used here rather than a local import in pyplot?

Reply:
I don't think that really works, because in backend_bases you don't know what the correct new_figure_manager is (that's really a pyplot concept)?

Reply:
fair.